website/.gitmodules

```diff
@@ -0,0 +1,24 @@
+[submodule "chisel3"]
```

Comment:
needs to be refactored, these submodules already exist

Reply:
This may not work correctly. If we want to maintain theses as submodules, I think the `.gitmodules` file needs to live in the root?